sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/implementation/DocumentsImpl.java

```diff
@@ -40,9 +40,10 @@
 import com.azure.search.documents.implementation.models.SuggestDocumentsResult;
 import com.azure.search.documents.implementation.models.SuggestOptions;
 import com.azure.search.documents.implementation.models.SuggestRequest;
+import reactor.core.publisher.Mono;
+
```

Comment:
Could you turn off the code reformatting?

Reply:
Will revert this change.